diagnostic.md

```diff
@@ -35,11 +44,16 @@ Write your answers inside this file, where it's indicated by the comments.
     directory and named `diagnostic.md`.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    git add ., but recommended not to do that by instructor(s)
```

Comment:
That's right, please do not do `git add .` if you can help it. You might accidentally sweep things into your commits that shouldn't be there.